docs/form-audit-log.rst

```diff
@@ -79,7 +79,7 @@ For the most accurate locations, set ``location-priority`` to `high-accuracy`. F
 
   Disabling location tracking will not prevent users from filling out forms, but these changes are logged as events in the log.
 
-.. _viewing-audit-logs:
+.. _form-audit-log-change-tracking:
```

Comment:
Not consistent with previous tag of `_audit-geolocation-tracking`. Maybe change that or change these ones you've added?